Monika After Story/game/script-topics.rpy

```diff
@@ -15751,3 +15752,33 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_kamige",
+            category=['games'],
+            prompt="What is kamige?",
+            pool=True,
+            unlocked=False,
+            rules={"no unlock":None}
+        )
+    )
+
+label monika_kamige:
+    m 1euc "Oh, that's right... it's not exactly a common term."
+    m 4eub "{i}Kamige{/i} is a Japanese slang word that is mostly used by visual novel fans."
+    m 4ruc "If I were to try to translate it...I think it would be something like {i}godly game.{/i}"
+    m 2eub "It's sort of like when people talk about their favorite classic books or movies."
+    m 2hksdlb "I was kind of joking when I said it about this game, but it {i}did{/i} seem to get very popular for some reason."
+
```

Comment:
This heavily contradicts 'monika_archetype', I suggest removing these 4 lines entirely.

Reply:
I was unsure if I wanted to keep the doki-cares variant, since monika_archetype is locked if you have that persistent variable. I think you're right that it's still too contradictory to keep in. I'll rewrite or delete them in the near future.